.travis.yml

```diff
@@ -68,6 +68,7 @@ install:
     - echo "memory_limit=2048M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
     - php ./bin/fix-symfony-version.php "$SYMFONY_VERSION"
     - composer install
+    - sed -i 's/525568/16777471/' vendor/kwn/php-rdkafka-stubs/stubs/constants.php
```

Comment:
this is needed for the tests, since the stub has a really old version

Reply:
A comment near the line would be great.